### DISCUSSION TITLE
Build result in dist folder from tailwindcss + vite. Does not appear as in the development process

### DISCUSSION THREAD

**Discussion by mir161803** [tailwindlabs/tailwindcss#4465] [Help]
Hello, I made a sample website using tailwindcss + vite like a video tutorial that I watched on youtube. Everything went well at first, but when it comes to the optimization step with the "vite build". The build results in the dist folder produce a website that doesn't match the website version in development. I have confirmed that the settings for purge are correct and if you want to see the results from the dist folder, you can visit my github repo at [this link](https://github.com/Nassardhi/tailwind). Is this problem because I missed some steps?

**Comment by simonswiss** [comment]
Hey!

I just cloned your repo, and when I build the project with the `npm run build` command, the project looks as it should:

![image](https://user-images.githubusercontent.com/485747/119619210-176e6b80-be47-11eb-9979-20dca4730316.png)

What's probably happening is you're trying to preview the site by just opening the file in your browser? Since the generated `index.html` file in the `dist` folder has a `<link>` tag pointing to a relative path...

```html
<link rel="stylesheet" href="/assets/index.77a7d2f1.css">
```

... you need to visit that page from a context where the `dist` directory is the `root` path.

You can use a simple local server like [`live-server`](https://www.npmjs.com/package/live-server) or [`http-server`](https://www.npmjs.com/package/http-server) to achieve that:

```sh
live-server dist
```

That will open the `index.html` page inside the `dist` directory on `http://localhost:8080/`, and you should see the website displayed just fine, with the CSS properly purged 👍
